ref/general/requestTiming.adoc

```diff
@@ -0,0 +1,71 @@
+:page-layout: general-reference
+:page-type: general
+:page-description: The request timing feature automatically detects slow requests and logs the information when the request is completed.
+:page-categories: MicroProfile
+:seo-title: The request timing feature automatically detects slow and hung requests
+:seo-description: The request timing feature automatically detects slow and hung requests and logs the information when the request is completed.
+= Slow and hung request detection
+
+Slow or hung requests can happen, and your application needs a continuous monitoring tool to detect the cause.
```

Comment:
the first sentence seems fragmented, and doesn't tell you what a slow or hung request is.

"When an application takes longer than expected to complete a request, the request can be logged as slow or hung. Your applications need a continuous monitoring tool to detect and analyze slow and hung requests."

or something like that....